modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts

```diff
@@ -177,7 +142,7 @@ const StyleSheetLoader = (documentOrShadowRoot: Document | ShadowRoot, settings:
       }
 
       link = linkElem.dom;
-      link.onload = waitForWebKitLinkLoaded;
+      link.onload = passed;
```

Comment:
Do we need to add any additional tests for this? For example, with failing / passing stylesheets?

Reply:
I thought we already had tests for this?